backend/app/views/surveys/index.slim

```diff
@@ -8,6 +8,9 @@
               = survey[:name]
             p.card-text
               = survey[:description]
+            p.card-text
+              =  question_types(survey).join(", ")
+              |  type of questions.
             p.card-text
               =  t('surveys.index.question', count: survey[:answered_questions_quantity])
               |  answered.
```

Comment:
I know this is not related to your PR, but please translate also

Reply:
@edimossilva the type of the question is translated, in `SurveyHelper`. 

So you mean only this text right?
![image](https://user-images.githubusercontent.com/79217155/151032302-b5bf54bf-9345-4723-a7f8-2e4addb992c5.png)

Reply:
yes!

Reply:
also this one ` |  answered.`

Reply:
okay